Combine long press and two button reset (#153)
NordicSemiconductor/pc-nrfconnect-npm: Changelog.md

## 1.2.2 - UNRELEASED

### Changed

-   Updated Firmware to version 1.1.1+0

### Removed

> -   Removed the separate toggle for two button reset. This functionality Will be
>     incorporated into the long press configuration instead.

Reviewer: ```suggestion
-   Removed the separate toggle for the two-button reset. This functionality has been incorporated into the long-press configuration instead.
```
Will be? This PR suggests it is already incorporated...
